ns/namespace.go

```diff
@@ -94,12 +94,69 @@ func FromCompositionTags(object interface{}, current string, namespace *[]string
 		InspectEmptyContainers(AlwaysFalse))
 }
 
-// GetValueByNamespace returns value stored in struct composition.
+// GetValueByNamespace returns value stored in nested map, array or struct composition.
+func GetValueByNamespace(object interface{}, ns []string) interface{} {
+
+	if len(ns) == 0 {
+		fmt.Fprintf(os.Stderr, "Namespace length equal to zero\n")
```

Comment:
Changing signature of this function to introduce additional error handling would require changes in plugins which are using it. I would like it as is, plugin author can handle `nil` values which this function returns on failure.
As for `panic` this is really bad idea to crash whole plugin on problem with getting value for requested metric.